homework_1/task_02/taxes/taxes.go

```diff
@@ -0,0 +1,69 @@
+package taxes
+
+import (
+	"github.com/pkg/errors"
+)
+
+type TaxClass struct {
+	UpperAmount  float64
+	Percentage float64
+}
+
+
+func checkLevels(taxClasses []TaxClass) bool {
```

Comment:
`validateTaxClasses` might be better.

Also - return `error` instead of a `bool` here. That way you can more easily describe what went wrong during the validation.